### PR TITLE
feat(CX-3598): add new onboarding cards to home screen

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-3598] 

### Description

https://user-images.githubusercontent.com/36167539/231513604-9d69b90b-cf7e-4707-8009-69b24ba3f216.mp4

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- added new Artsy onboarding rail to the home screen -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3598]: https://artsyproduct.atlassian.net/browse/CX-3598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Home/useHomeModules.ts [hunk 1]
```diff
@@ -58,6 +60,14 @@ export const useHomeModules = (props: HomeProps, cards: ReactAppboy.CaptionedCon
         title: "Auctions",
         type: "sales",
       },
+      {
+        data: true, // displays static data
```
Comment (olerichter00) [inline src/app/Scenes/Home/useHomeModules.ts:64]: **question:** Do we need to pass anything here or can we leave the data empty?

```suggestion
```

**Review by olerichter00 — APPROVED** [review]
Looks great 🌟 

I wonder if we could or need to reduce the size of the images 🤔

File: src/app/Scenes/Home/useHomeModules.ts (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/Home/useHomeModules.ts:64]: Data is the required prop and in the Home.ts file, we check the data https://github.com/artsy/eigen/blob/main/src/app/Scenes/Home/Home.tsx#L208 and display the component only if the data is present.
I thought the way I did it is the easiest

Comment (olerichter00) [reply src/app/Scenes/Home/useHomeModules.ts:64]: That makes sense 👍 Let's keep it as it is.

**Comment by dariakoko** [comment]
@olerichter00 You mean the images themselves, not the way we display them in the component? Not sure, I downloaded them from figma and they have the same size as the ones on the old cards. You would suggest reducing the size?

**Comment by olerichter00** [comment]
> @olerichter00 You mean the images themselves, not the way we display them in the component? Not sure, I downloaded them from figma and they have the same size as the ones on the old cards. You would suggest reducing the size?

Ok, let's add them as they are. I remember @MounirDhahri was working on the image sizes in Eigen a while ago. @MounirDhahri, do you think it is possible and needed to reduce the size of the images?

**Comment by dariakoko** [comment]
@MounirDhahri @olerichter00 maybe we could document somewhere the optimal image sizes? Do we have a readme for similar purposes?

**Comment by MounirDhahri** [comment]
I don't think it's possible to document that in numbers but it's probably a good idea to write this somewhere in our docs. The rationale behind it comes from some studies that emerged a few years ago about how [app size matters](https://medium.com/googleplaydev/shrinking-apks-growing-installs-5d3fcba23ce2). Overall, the smaller the app size, the more people convert into installing the app. In our case, since we have a fair amount of images in the app, our bundle size has been increasing quickly and it's important for us to try to cut the image size to avoid [affecting our conversion](https://segment.com/blog/mobile-app-size-effect-on-downloads/)

**Pull request merged**